### ISSUE TITLE
ClimateTrace Data Pipeline

### ISSUE THREAD

**Issue by caldeirav** [os-climate/os_c_data_commons#233]

**Comment by caldeirav** [comment]
Data pipeline for ClimateTrace (emissions by sector): https://climatetrace.org/downloads

Methodology including source data used in computation is available at https://github.com/climatetracecoalition

@MichaelTiemannOSC could you comment on usefulness of this data in OS-Climate context (any stream this would be interesting for?). Limitation of this approach is we would not get lineage from computations done by ClimateTrace.

**Comment by MichaelTiemannOSC** [comment]
I don't think we need computational lineage to start.  Federating the computed data we can compare/contrast with ESSD and others.  If, down the road, Climate trace wants to make deeper data public (possibly using the Data Commons architecture), so much the better.
